src/main/java/com/laborhelper/enterprise/dto/LaborAction.java

```diff
@@ -6,7 +6,7 @@
 import java.util.Map;
 
 public @Data class LaborAction {
-    @NonNull private String id;
+    @NonNull private int id;
```

Comment:
I would recommend changing the id data type to int so you can use it in a DAO hashmap.

Reply:
I appreciate your response and looking over my pull request. It's definitely correct to leave it as String if you don't know the DAO implementation yet. I just changed it to int so it would work with the DAO stub I've included in the pull request. I just used int as my preference since it's easy to use in this context, that's all. You can change it to whatever you want, if the professor and you decide to merge this request. Have a good weekend!